cfme/base/__init__.py

```diff
@@ -211,6 +229,9 @@ def get_master(self):
 
         return self.instantiate(sid=server.id)
 
+    def get_current(self):
```

Comment:
I'd like the answer to be a docblock comment please.

Reply:
And in general, a docblock explanation of how this differs from get_master.

I would think that this would have to look at what endpoint the browser is using as well, to know if the current browser session is on the server matching the appliance.server_id() value.

Reply:
Removed get_current() and just used appliance.server in test_paginator.py.